R/qenv-eval_code.R

```diff
@@ -53,48 +53,44 @@ setMethod("eval_code", signature = c(object = "qenv.error"), function(object, co
     return(object)
   }
   code_split <- split_code(paste(code, collapse = "\n"))
+
   for (i in seq_along(code_split)) {
```

Comment:
I think we can evluate all the code split at once with this package.

It already gives the warnings / errors / outputs line by line.

We just need to map it after the fact, that is, outputs, errors, messages and warnings after source element are mapped to that source.